app/src/main/java/org/greenstand/android/TreeTracker/dashboard/DashboardScreen.kt

```diff
@@ -63,13 +61,30 @@ fun DashboardScreen(
     val state by viewModel.state.observeAsState(DashboardState())
     val snackBar by viewModel.snackBar.observeAsState()
     val navController = LocalNavHostController.current
+    var dialogOpened by remember { mutableStateOf(false) }
+    var showDialog by remember { mutableStateOf(false) }
 
+    if (showDialog){
+        DisplayAlertDialog(
+            title = "Upload Trees Soon",
+            message ="You have over 2000 trees, please upload as soon as you can.",
```

Comment:
Text that the user sees should always use string resources to allow translations.